include/acl_selectors.php

```diff
@@ -432,6 +442,7 @@ function acl_lookup(App $a, $out_type = 'json') {
 		$sql_extra = "AND `name` LIKE '%%".dbesc($search)."%%'";
 		$sql_extra2 = "AND (`attag` LIKE '%%".dbesc($search)."%%' OR `name` LIKE '%%".dbesc($search)."%%' OR `nick` LIKE '%%".dbesc($search)."%%')";
 	} else {
+		/// @TODO Avoid these needless else blocks by putting variable-initialization atop of if()
```

Comment:
I really see it differently. I like to assign variables on the same level if possible.